### ISSUE TITLE
More Broken Missing Files

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#9745]
**Describe the bug**
Now the encoder seems to be failing on every file.

**To Reproduce**
1. Go to encode a file
2. Observe it is Broken Missing Files always
3. Scan the logs for errors
4. Find errors on the encoder like:

Note: I have multiple "encoders" (i.e. copies of the encoder in /var/www/html like /var/www/html/encoder1 /var/www/html/encoder2) which exhibit the same behavior. I suspect it may be an AVideo permissions issue?

```
"response_raw":"< !DOCTYPE HTML PUBLIC \"-\/\/IETF\/\/DTD HTML 2.0\/\/EN\" >\n< html >< head >\n< title >403 Forbidden< \/title >\n< \/head >< body >\n< h1 >Forbidden< \/h1 >\n< p >You don't have permission to access this resource.< \/p >\n< \/body >< \/html >\n",
"response":null,
"msg":"Response was not an json object"} debug_backtrace=[{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php",
"line":1825,

```
**Expected behavior**
Not to get a 403 accessing the encoder

**Error Logs**

`[26-Dec-2024 07:25:13 UTC] [2024-12-26 07:25:13] sendToStreamer https://conspyre.tv/aVideoEncoder.json in 0.011 seconds {"error":true,"target":"aVideoEncoder.json","postFields":{"duration":"0:02:08","title":"This guys got a good point- \ud83e\udd23\ud83d\udd25","first_request":1,"categories_id":0,"format":"mp4","resolution":"720","videoDownloadedLink":"","description":"","downloadURL":"https:\/\/phoenix.conspyre.tv\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4","chunkFile":4,"encoderURL":"https:\/\/phoenix.conspyre.tv\/","keepEncoding":"0","video":"32.59MB","streamers_id":"1","user":"Conspyre","pass":"_user_hash_Nzl3cG51TUI4R20ydlpMbTc0MEhrRlZkUGVFUTlGRmxLK0l6cnIvUW9DM21KaHFHUXlQM0pic3RFUlVTREFIMk9VRTRjcHNlM05xeVRLajA0Yi8yeGc9PQ==","return_vars":"{\"videos_id\":19802,\"video_id_hash\":\"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=\",\"releaseDate\":\"2024-12-25 23:08:00\"}","releaseDate":"2024-12-25 23:08:00","releaseTime":1735168080,"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","timezone":"UTC"},"response_raw":"<!DOCTYPE HTML PUBLIC \"-\/\/IETF\/\/DTD HTML 2.0\/\/EN\">\n<html><head>\n<title>403 Forbidden<\/title>\n<\/head><body>\n<h1>Forbidden<\/h1>\n<p>You don't have permission to access this resource.<\/p>\n<\/body><\/html>\n","response":null,"msg":"Response was not an json object"} debug_backtrace=[{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php","line":1825,"function":"sendToStreamer","class":"Encoder","type":"::","args":["aVideoEncoder.json",{"duration":"0:02:08","title":"This guys got a good point- \ud83e\udd23\ud83d\udd25","first_request":1,"categories_id":0,"format":"mp4","resolution":"720","videoDownloadedLink":"","description":"","downloadURL":"https:\/\/phoenix.conspyre.tv\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4","chunkFile":4,"encoderURL":"https:\/\/phoenix.conspyre.tv\/","keepEncoding":"0","video":{"name":"\/var\/www\/html\/phoenix\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4","mime":"","postname":""},"streamers_id":"1","user":"Conspyre","pass":"_user_hash_Nzl3cG51TUI4R20ydlpMbTc0MEhrRlZkUGVFUTlGRmxLK0l6cnIvUW9DM21KaHFHUXlQM0pic3RFUlVTREFIMk9VRTRjcHNlM05xeVRLajA0Yi8yeGc9PQ==","return_vars":"{\"videos_id\":19802,\"video_id_hash\":\"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=\",\"releaseDate\":\"2024-12-25 23:08:00\"}","releaseDate":"2024-12-25 23:08:00","releaseTime":1735168080,"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","timezone":"UTC"},{"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","releaseDate":"2024-12-25 23:08:00"},{}]},{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php","line":1925,"function":"sendFile","class":"Encoder","type":"::","args":["\/var\/www\/html\/phoenix\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4",{"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","releaseDate":"2024-12-25 23:08:00"},"mp4",{},"720",4]},{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php","line":1920,"function":"sendFileChunk","class":"Encoder","type":"::","args":["\/var\/www\/html\/phoenix\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4",{"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","releaseDate":"2024-12-25 23:08:00"},"mp4",{},"720",4]},{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php","line":1920,"function":"sendFileChunk","class":"Encoder","type":"::","args":["\/var\/www\/html\/phoenix\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4",{"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","releaseDate":"2024-12-25 23:08:00"},"mp4",{},"720",3]},{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php","line":1920,"function":"sendFileChunk","class":"Encoder","type":"::","args":["\/var\/www\/html\/phoenix\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4",{"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","releaseDate":"2024-12-25 23:08:00"},"mp4",{},"720",2]},{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php","line":1614,"function":"sendFileChunk","class":"Encoder","type":"::","args":["\/var\/www\/html\/phoenix\/videos\/avideoTmpFile_824_streamers_id_1_720.mp4",{"videos_id":19802,"video_id_hash":"WlZoNnQ4NytJSHpFZGtjbWZIUE1tTVIzK2k2MmM0eU9VallBNXhJdm5Scz0=","releaseDate":"2024-12-25 23:08:00"},"mp4",{},"720"]},{"file":"\/var\/www\/html\/phoenix\/objects\/Encoder.php","line":1378,"function":"send","class":"Encoder","object":{},"type":"->","args":[]},{"file":"\/var\/www\/html\/phoenix\/view\/run.php","line":7,"function":"run","class":"Encoder","type":"::","args":[]}]`

**Comment by DanielnetoDotCom** [comment]
Clear that is some configuration on your Streamer

check this 

![image](https://github.com/user-attachments/assets/c0766e10-5403-4edb-bb0f-c037a22ef666)

now check how it should look like

![image](https://github.com/user-attachments/assets/a8e0e28d-6a30-4d01-a8f2-14e1b9bf1b0d)

**Comment by walt93** [comment]
Daniel, not sure what to check here. 

I've checked out a new AVideo directory, and copied the plugins and videos directory from the old site into it - and deployed it. 

The resulting site has the same issue, forbidden accessing aVideoEncoder.json.

Is this a file permissions issue? What are the correct permissions (if I could ssh into the demosite and check it, I would...)

**Comment by DanielnetoDotCom** [comment]
this is something on your apache configuration, not sure where.

**Comment by walt93** [comment]
Yeah, I spent a few hours debugging this last night. Spun up new avideo site, same issue occurs there too. Apache is blocking the .json at access_compat. I was able to "make it work" by duplicating the endpoint as aVideoEncoder_test (instead of .json) and modifying the Encoder.php script to call the _test route instead of .json which is obviously a gross hack.

Going back into the trenches to figure out whats going on with Apache.
